webapp/.gitignore

```diff
@@ -11,3 +11,7 @@
 /phpunit.xml
 .phpunit.result.cache
 
+
+###> lexik/jwt-authentication-bundle ###
+/config/jwt/*
```

Comment:
why do you need that? Does the Symfony app store JWTs on the disk?

Reply:
This configuration is used by the JWT library to store the RSA key pair used to sign the tokens